secrets.tf

```diff
@@ -56,3 +68,17 @@ resource "kubernetes_secret" "database_secret_name" {
     "password" = local.map_of_output_drupal_databases[each.key].password
   }
 }
+
+resource "kubernetes_secret" "redis_secret_name" {
```

Comment:
Use `kubernetes_secret_v1` resource type.

Why the name for this resource is `redis_secret_name`?